test/simplegraphs/generators/randgraphs.jl

```diff
@@ -364,4 +364,24 @@
         @test isvalid_simplegraph(rog3)
         @test !is_cyclic(rog3)
     end
+
+    @testset "randbn" begin
+        for i in 0:10
+            @test Graphs.SimpleGraphs.randbn(i, 0.0; rng=rng) == 0
+            @test Graphs.SimpleGraphs.randbn(i, 1.0; rng=rng) == i
+        end
+        N = 30
+        s1 = zeros(N)
+        s2 = zeros(N)
+        for i in 1:N
+            s1[i] = Graphs.SimpleGraphs.randbn(5, 0.3)
+            s2[i] = Graphs.SimpleGraphs.randbn(3, 0.7)
+        end
+        μ1 = mean(s1)
+        μ2 = mean(s2)
+        sv1 = std(s1)
+        sv2 = std(s2)
+        @test μ1 - sv1 <= 0.3 * 5 <= μ1 + sv1 # since the stdev of μ1 is around sv1/sqrt(N), this should rarely fail
+        @test μ2 - sv2 <= 0.7 * 3 <= μ2 + sv2
```

Comment:
Interesting approach, so we are using a confidence interval to check if the mean of the distribution is within that interval?

I am not sure what kind of confidence level we get here and if it is really enough, but as the rng is probably seeded and the tests pass, this seems to be ok.

Reply:
I was also initially puzzled, that's why I suggested to add a comment. 
Now that I'm thinking about it again, multiplying all sides by $N$ we get that the bound is testing if the sum of all Bernoulli r.v.s in all experiments is within $\Theta(N)$, while a Chernouff bound should imply that they are within $O(\sqrt{N\log N})$ with high probability, so the probability is exponentially good. 
The way it is written is a bit obscure but I don't have a clearer alternative from the top of my head.

Reply:
@simonschoelly regarding this point, I thought that Aurora had copied the approach from elsewhere in the library. 
Instead, she told me that she introduced it herself.
I did find similar tests in the past, so we will try to reformulate it more clearly.